trunk/SUAVE/Methods/Performance/estimate_landing_field_length.py

```diff
@@ -51,7 +51,7 @@ def estimate_landing_field_length(vehicle,analyses,airport):
     # Unpack
     # ==============================================
     atmo            = airport.atmosphere
-    altitude        = airport.altitude * Units.ft
```

Comment:
actually this probably shouldn't have units

Reply:
Changes reverted